projects/plugins/jetpack/changelog/update-story-block-keyboard-shortcuts

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: compat
+
+Story Block: Improve accessibility and resolve z-index issues when playing in fullscreen.
```

Comment:
We've been leaving the Story block out of the release notes since it's not available on the web yet and untestable. You can update the changelog such that the comment won't be added to the release notes by using the Comment field, like I did here: https://github.com/Automattic/jetpack/pull/19176/files

Also we'd want to update the PR description not to have release notes.